src/pages/ScholarshipForm.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import { Container, Button, TextField } from '@material-ui/core';
+
+function ScholarshipForm() {
+  const [formFieldStates, setFormFieldStates] = useState({
```

Comment:
Could this state could be initialized to an empty object or just an object with the deadline?

Reply:
It gives me the following warning: `Warning: A component is changing an uncontrolled input of type text to be controlled.`